internal/pkg/externalplugins/config.go

```diff
@@ -15,6 +15,11 @@ const (
 	defaultGracePeriodDuration = 5
 )
 
+var AllowActions = []string{
```

Comment:
Better make it be a const.

Reply:
Slice doesn't seem to be a constant.

Reply:
It can be two consts, and since we won't be modifying it, using var is not appropriate in this place.